charts/fluid-dataprocess/common/templates/configmap.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-scripts
+  labels:
+    release: {{ .Release.Name }}
```

Comment:
I think the ownerReference may help a lot.

Reply:
Done

Reply:
Done.